packages/node_modules/@node-red/nodes/locales/en-US/messages.json

```diff
@@ -853,7 +853,7 @@
             "appendedfile": "appended to file: __file__"
         },
         "encoding": {
-            "none": "None",
+            "none": "utf8[default]",
```

Comment:
definitely better... though does it also handle binary files by autodetect ? maybe "utf8/binary [default]"  or "auto utf8/binary [default]"     (sorry to be picky, still trying to get it right)

Reply:
I think binary files should be read in binary buffer.  In this case, no conversion is performed.  If it is read in a string, file contents is interpreted as utf8 sequence.

Reply:
Still not sure this feels right.  I'm thinking maybe ` Auto [utf8/binary] ` may be better ?  Maybe with a comment in the info text to say "File encoding allows fixing the encoding of files to a variety of common formats. The default Auto mode tries to detect if the content is UTF8, and if not detected uses binary encoding.

Reply:
Hi, following is an excerpt from current (old) `file in` node code.
Because no encoding option is specified for `createReadStream` (a), file contents are read in to an `Buffer` as binary sequence at (b).  
When reached end of file,  an accumulated binary `Buffer` is converted to string at (c) by `toString` function with default encoding (`utf8`; please refer [here](https://nodejs.org/api/buffer.html#buffer_buf_tostring_encoding_start_end) ) if target is string.  
If target is binary `Buffer`, the buffer is assigned to `msg.payload` without conversion at (d).
Since encoding conversion take effect only for string case, default behavior is `utf8` interpretation.
```
var rs = fs.createReadStream(filename) // (a)
    .on('readable', function () {
        ...
        while (null !== (chunk = rs.read())) { // (b)
            if (node.chunk === true) {
                ...
            }
            else {
                lines = Buffer.concat([lines,chunk]);
            }
        }
    })
    ...
    .on('end', function() {
        if (node.chunk === false) {
            if (node.format === "utf8") { msg.payload = lines.toString(); } // (c)
            else { msg.payload = lines; } // (d)
            node.send(msg);
        }
    }
    ...
```

Reply:
Apologies - yes you are correct. I was thinking of the file out node.

Reply:
No problem.  thank you for clarifying this.
Regarding `file out` node, it takes similar path in reverse order.

Reply:
true - the existing default encoding for write is `binary` isn't it...

Reply:
yes. file contents are read into binary `Buffer` without conversion, then interpreted as `utf8` string by `toString` function.

Reply:
OK - decision time...  

We think it should just be labelled   "Default"   
(which is what it is.. :-)... as we can't decide how best to describe it as "the way it used to work which was actually binary, but handled text just fine, as long as it was simple English (ascii)" - so lets just call it "Default" and get this thing merged

once again apologies for dithering so much

Reply:
hi. updated the label to "Default" according to your suggestion.